models/reviews.models.js

```diff
@@ -1,30 +1,54 @@
 const db = require('../db/connection.js');
 const { checkReviewIdExists } = require('../myUtils/myUtils.js');
 
-exports.selectReviews = () => {
-  return db
-    .query(
-      `SELECT reviews.review_id, title, category, designer, owner, review_body, review_img_url, reviews.votes, reviews.created_at, COUNT(comments.review_id) ::INT AS comment_count
-    FROM reviews
-    JOIN users ON reviews.owner = users.username 
-    LEFT JOIN comments ON reviews.review_id = comments.review_id
-    GROUP BY reviews.review_id, users.username
-    ORDER BY reviews.created_at DESC;`
-    )
-    .then(reviews => {
-      return reviews.rows;
-    });
+exports.selectReviews = (category, sort_by = 'created_at', order = 'DESC') => {
+  const validColumns = [
+    'title',
+    'designer',
+    'owner',
+    'category',
+    'created_at',
+    'votes',
+  ];
+
+  const validSorts = ['DESC', 'ASC'];
+
```

Comment:
Do you have anything that validates the value for ORDER BY to prevent injection? I can you do for SORT BY

Reply:
Yeah it was just above it

Reply:
Also - thank you!